custom/ewsghana/api.py

```diff
@@ -132,6 +166,14 @@ class EWSApi(APISynchronization):
     ]
     PRODUCT_CUSTOM_FIELDS = []
 
+    def apis(self):
```

Comment:
@agrzywinski perhaps we can just leave the extensions commented out but merge everything else? that way there will be less likelihood of merge conflicts when it is time to turn them on.